docroot/modules/custom/va_gov_post_api/src/Service/PostFacilityService.php

```diff
@@ -708,31 +726,49 @@ protected function setServiceTerm() {
   /**
    * Log service by facility.
    *
+   * @param string $logFile
+   *   Path to the log file.
    * @param string $facilityApiId
    *   Facility API Id.
-   * @param \Drupal\Core\Entity\EntityInterface $facilityService
+   * @param string $facilityService
    *   Facility service.
    */
-  protected function logService(string $facilityApiId, EntityInterface $facilityService) {
-    $log_message = date('Y-m-d H:i:s') . '|' . $facilityApiId . '|' . $facilityService->title->value . "\n";
-    $handle = fopen(self::$logFile, "a");
-    fwrite($handle, $log_message);
-    fclose($handle);
+  protected function logService(string $logFile, string $facilityApiId, string $facilityService) {
+    $log_message = date('Y-m-d H:i:s') . '|' . $facilityApiId . '|' . $facilityService . "\n";
```

Comment:
```suggestion
    $log_message = date('Y-m-d H:i:s') . "|{$facilityApiId}|{$facilityService}\n";
```
Easier to read, and faster as it only calls the concatenator once, rather than for each '.'